Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -674,7 +682,9 @@ module Kitos.Utility.KendoGrid {
 
         saveGridForOrganization() {
             if (confirm(`Er du sikker på at du vil gemme nuværende kolonneopsætning af felter som standard til ${this.user.currentOrganizationName}`)) {
-                this.gridState.saveGridOrganizationalConfiguration(this.gridBinding.mainGrid, this.overviewType);
+                this.gridState
+                    .saveGridOrganizationalConfiguration(this.gridBinding.mainGrid, this.overviewType)
+                    .then(() => this.reload());
```

Comment:
I had to reload the page after saving, because otherwise I couldn't get the "delete filter" enabled update to work.

The delete filter "button" would work, but the css class (line 1155) wouldn't change, and the text would remain gray